examples/api/test/material/app_bar/sliver_app_bar.2_test.dart

```diff
@@ -35,9 +33,15 @@ void main() {
 
     expect(actualTitleStyle, expectedTitleStyle);
 
-    expect(tester.getBottomLeft(find.text(title).first).dy, 96.0);
-    await tester.drag(find.text(title).first, _kOffset, touchSlopY: 0, warnIfMissed: false);
+    // Scrolling the screen moves the title up.
+    expect(tester.getBottomLeft(titleFinder).dy, 96.0);
+    await tester.drag(
+      titleFinder,
+      const Offset(0.0, -200.0),
+      touchSlopY: 0,
+      warnIfMissed: false);
+
```

Comment:
Any particular reason for setting `touchSlopY` and `warnIfMissed`? (I tested without and the test still pass).

```suggestion
    // Scrolling the screen moves the title up.
    expect(tester.getBottomLeft(titleFinder).dy, 96.0);
    await tester.drag(titleFinder, const Offset(0.0, -200.0));
```